accelerate/nextjs-starter/lib/db.ts

```diff
@@ -0,0 +1,20 @@
+import { PrismaClient } from "@prisma/client/edge";
+import { withAccelerate } from "@prisma/extension-accelerate";
+
+// Learn more about instantiating PrismaClient in Next.js here: https://www.prisma.io/docs/data-platform/accelerate/getting-started
```

Comment:
I think this is an incorrect link.

Probably meant to be https://www.prisma.io/docs/orm/more/help-and-troubleshooting/help-articles/nextjs-prisma-client-dev-practices ?

Reply:
```suggestion
// Learn more about instantiating PrismaClient in Next.js here: https://www.prisma.io/docs/orm/more/help-and-troubleshooting/help-articles/nextjs-prisma-client-dev-practices
```